apps/web/src/server/views/applications/case-documentation/documentation-properties.njk

```diff
@@ -1,113 +1,93 @@
 {% extends "../layouts/applications.layout.njk" %}
-
 {% from "govuk/components/breadcrumbs/macro.njk" import govukBreadcrumbs %}
 {% from "govuk/components/details/macro.njk" import govukDetails %}
 {% from "govuk/components/tabs/macro.njk" import govukTabs %}
 {% from "govuk/components/summary-list/macro.njk" import govukSummaryList %}
 {% from "govuk/components/button/macro.njk" import govukButton %}
-
-
 {% block beforeContent %}
-	{{ govukBackLink({
+    {{ govukBackLink({
 		text: "Back",
 		href: breadcrumbItems[breadcrumbItems.length - 1].href
 	}) }}
 {% endblock %}
-
 {% block pageHeading %}{% endblock %}
-
 {% block pageContent %}
-	<div class="govuk-grid-row">
-		<div class="govuk-grid-column-full pins-dashboard-list--hide-controls govuk-!-margin-top-0">
-			{% block sectionContent %}
-				{{ govukBreadcrumbs({
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-full pins-dashboard-list--hide-controls govuk-!-margin-top-0">
+            {% block sectionContent %}
+                {{ govukBreadcrumbs({
 					classes: 'govuk-!-margin-top-0',
 					items: breadcrumbItems
 				}) }}
-
-
-				<h2 class="govuk-caption-xl">Document properties</h2>
-				<h3 class="govuk-heading-xl">{{ documentationFile.documentName|documentName }}</h3>
-
-				<ul class="govuk-list">
-					<li class="govuk-!-margin-top-7">{# documentationFile.description #}</li>
-					<li class="govuk-!-margin-top-4">Case reference: {{ case.reference }}</li>
-					<li class="govuk-!-margin-top-4">Document reference number:</li>
-					<li class="govuk-!-margin-top-4">
-						{{ documentationFile.type|fileType }}, {{ documentationFile.size|fileSize }}
-					</li>
-				</ul>
-
-				<div class="display--flex govuk-!-margin-top-7 govuk-!-margin-bottom-7">
-					<a class="govuk-button govuk-!-margin-0"
-					   href="upload/new/version">
-						Upload new version
-					</a>
-					<a class="govuk-button govuk-button--secondary govuk-!-margin-0 govuk-!-margin-left-4"
-					   href="{{ 'document-download'|url({caseId: caseId, documentGuid: documentationFile.guid}) }}">
-						Download
-					</a>
-				</div>
-
-				<nav>
-					{% set actionLinks %}
-						<a class="govuk-link display--block" href="/open">Open</a>
-
-						<a class="govuk-link display--block" href="/move">Move</a>
-
-						<a class="govuk-link display--block" href="{{ 'document'|url({caseId: caseId, folderId: folderId, documentGuid: documentationFile.guid, step: 'delete'}) }}">Delete</a>
-					{% endset %}
-					{{ govukDetails({
+                <h2 class="govuk-caption-xl">Document properties</h2>
```

Comment:
do you need an H1 on the page or is it inherited from the template

Reply:
no, H1 was missing. replaced the couple h2/h3 with the standard h1+caption